doc/azure_ci.md

```diff
@@ -30,14 +30,13 @@ used on your projects:
   with just the settings present in your `stack.yaml` file.
 * The complex Azure configuration is intended for projects that need
   to support multiple GHC versions and multiple OSes, such as open
-  source libraries to be released to Hackage. It tests against
-  cabal-install, as well as Stack on Linux and macOS. The
-  configuration is significantly more involved to allow for all of
-  this branching behavior. In the provided template, we use the Linux
-  build configuration to test against various combinations of
-  cabal-install and stack. The macOS and Windows configuration is used
-  only for testing Stack build configuration. These are the files for
-  the complex configuration:
+  source libraries to be released to Hackage. It tests against Stack
```

Comment:
I think the wording here can be simplified even further, can't it? We now have essentially identical scripts on Mac, Windows, and Linux, and test only against Stack, without any branching behavior.

Reply:
I have pushed a fix simplifying it further. Let me know if it needs further refining. Thanks!